assets/components/directDebit/directDebitForm/directDebitForm.jsx

```diff
@@ -0,0 +1,208 @@
+  updateAccountHolderConfirmation: (accountHolderConfirmation: boolean) => void,
+};
+/* eslint-enable react/no-unused-prop-types */
+
+// ----- Map State/Props ----- //
+
+function mapStateToProps(state) {
+  return {
+    isPopUpOpen: state.page.directDebit.isPopUpOpen,
+    sortCode: state.page.directDebit.bankSortCode,
+    accountNumber: state.page.directDebit.bankAccountNumber,
+    accountHolderName: state.page.directDebit.accountHolderName,
+    accountHolderConfirmation: state.page.directDebit.accountHolderConfirmation,
+  };
+}
+
+function mapDispatchToProps(dispatch) {
+
+  return {
+    updateSortCode: (sortCode: string) => {
+      dispatch(updateSortCode(sortCode));
+    },
+    updateAccountNumber: (accountNumber: string) => {
+      dispatch(updateAccountNumber(accountNumber));
+    },
+    updateAccountHolderName: (accountHolderName: string) => {
+      dispatch(updateAccountHolderName(accountHolderName));
+    },
+    updateAccountHolderConfirmation: (accountHolderConfirmation: boolean) => {
+      dispatch(updateAccountHolderConfirmation(accountHolderConfirmation));
+    },
+  };
+
+}
+
+// ----- Component ----- //
+
+const DirectDebitForm = (props: PropTypes) => (
+  <div className="component-direct-debit-form">
+    <dd className="component-direct-debit-form__direct-debit-logo">
+      <img src="#" alt="The Direct Debit logo" />
+    </dd>
+
+    <SortCodeInput
+      value={props.sortCode}
+      onChange={props.updateSortCode}
+    />
+
+    <AccountNumberInput
+      onChange={props.updateAccountNumber}
+      value={props.accountNumber}
+    />
+
+    <AccountHolderNameInput
+      onChange={props.updateAccountHolderName}
+      value={props.accountHolderName}
+    />
+
+    <ConfirmationInput
+      checked={props.accountHolderConfirmation}
+      onChange={props.updateAccountHolderConfirmation}
+    />
+
+    <dt className="component-direct-debit-form__advance-notice__title">
+        Advance notice
+    </dt>
+    <dd className="component-direct-debit-form__advance-notice__content">
+      <div>
+        <p>The details of your Direct Debit instruction including payment schedule, due date,
+          frequency and amount will be sent to you within three working days. All the normal
+          Direct Debit safeguards and guarantees apply.
+        </p>
+        <p>
+          Your payments are protected by the <a target="_blank" rel="noopener noreferrer" href="https://www.directdebit.co.uk/DirectDebitExplained/pages/directdebitguarantee.aspx">Direct Debit guarantee</a>.
+        </p>
+        <div>
+          <div>The Guardian, Unit 16, Coalfield Way, Ashby Park, Ashby-De-La-Zouch, LE65 1JT
+            United Kingdom
+          </div>
+          <div>Tel: +44 (0) 330 333 6767</div>
+          <div><a href="mailto:support@theguardian.com">support@theguardian.com</a></div>
+        </div>
+      </div>
+    </dd>
+  </div>);
+
+// ----- Default Props ----- //
+
+DirectDebitForm.defaultProps = {
+
+};
```

Comment:
Empty?

Reply:
I am going to look deeper into the props (including props initial values) in the second phase of this feature development.